generators/server/templates/src/main/java/package/web/rest/UserResource.java.ejs

```diff
@@ -286,18 +292,34 @@ public class UserResource {
     <%_ if (databaseType === 'sql' || databaseType === 'mongodb' || databaseType === 'neo4j' || databaseType === 'couchbase') { _%>
         <%_ if (!reactive) { _%>
     public ResponseEntity<List<<%= asDto('User') %>>> getAllUsers(Pageable pageable) {
+        <%_ if (authenticationType !== 'oauth2') { _%>
+        if (!onlyContainsAllowedProperties(pageable)) {
+            return ResponseEntity.badRequest().build();
+        }
+        <% } %>
```

Comment:
should be changed to `<%_ } _%>`

Reply:
It's intended to let a blank line between the end of the if statement and the rest of the code.

Reply:
Better to use always same syntax <%_ } _%> and explicitly add a blank line